java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/JdbcToArrow.java

```diff
@@ -34,10 +34,10 @@
 
 /**
  * Utility class to convert JDBC objects to columnar Arrow format objects.
- * <p>
- * This utility uses following data mapping to map JDBC/SQL datatype to Arrow data types.
- * <p>
- * CHAR --> ArrowType.Utf8
+ *
+ * <p>This utility uses following data mapping to map JDBC/SQL datatype to Arrow data types.
```

Comment:
@jacques-n @siddharthteotia @icexelloss most of these changes seemed pretty standard. The paragraph tag was the only thing that might have some discussion. The rule is that the `<p>` tag be before the first word after any newlines, after the first paragraph. There are no other options to configure this, other than disabling, but it didn't seem like a big deal to me to follow this rule.

Reply:
Personally I prefer the old style for readability - the misalignment bothers me a bit.

I am slight +1 to disabling the <p> tag rule.

Reply:
I agree, it makes the code look a little misaligned but currently this is really inconsistent in our Javadocs. Sometimes there is no tag, sometimes 2 tags, etc.  And if there are no tags, then the compiled Javadoc collapses to one paragraph. So it seemed best to keep it so that the compiled Javadocs end up consistent. We could always disable in the future and change the tags with a global search since they will be consistent. WDYT?

Reply:
It seems there is an example here that allows <p> not followed by a word: 
http://checkstyle.sourceforge.net/config_javadoc.html#JavadocParagraph

"In case of tagImmediatelyBeforeFirstWord set to false the following example will not have any violations:"

```
/**
 * Some Javadoc.
 *
 * <p>
 * Some Javadoc.
 *
 * <p>  Some Javadoc.
 *
 * <p>
 * <pre>
 * Some preformatted Javadoc.
 * </pre>
 *
 */
```

I didn't try it myself, but does that allows the javadoc before?

Reply:
Yeah, it's strange but that's not a valid property. I try it and get:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (validate) on project arrow-java-root: Failed during checkstyle configuration: cannot initialize module TreeWalker - Property 'tagImmediatelyBeforeFirstWord' in module JavadocParagraph does not exist, please check the documentation
```
And I tried `allowNewlineParagraph` which is a valid property that sounds like it might do this, but didn't seem to have any effect.  I don't know, maybe a bug or issue with the checkstyle docs...

Reply:
I also tried searching around a bit, but didn't come up with anything

Reply:
Yeah, OK. Don't think this is a big issue.